frontend/src/views/Pages/Register.spec.js

```diff
@@ -105,6 +114,87 @@ describe('Register', () => {
       })
     })
 
-    // To Do: Test lines 156-197,210-213
+    describe('resetForm', () => {
+      beforeEach(() => {
+        wrapper.find('#registerFirstname').setValue('Max')
+        wrapper.find('#registerLastname').setValue('Mustermann')
+        wrapper.find('#Email-input-field').setValue('max.mustermann@gradido.net')
+        wrapper.find('input[name="form.password"]').setValue('Aa123456')
```

Comment:
Here is a bug that I don't understand! The password is not reseted

Reply:
Let us open a new issue for that